src/xmipp/bindings/python/xmipp_conda_envs.py

```diff
@@ -0,0 +1,22 @@
+XMIPP_CONDA_ENVS = {
+  "xmipp_DLTK_v0.3": {
+    "pythonVersion": "3",
+    "dependencies": ["pandas=0.23.4", "scikit-image=0.14.2", "opencv=3.4.2",
```

Comment:
these are minimal required versions?

Reply:
These are "frozen" requirements. All protocols were tested under these setup

Reply:
These are the version that will be installed. In other words, these are what defines the conda environment.

Reply:
At some moment, we should allow for flexible (at least) bug fix version

Reply:
This is not a problem. Just create another environ, like xmip_DLTK_devel_vXX and tell your script that it should use it instead

Reply:
I0d go for @DStrelak suggestion if I were sure that package maintainers follow semantic versioning scheme, which I'm sure in some cases they do not! I understand why freezing versions.

Reply:
Keep in mind that this 'dependencies' are not the requirements to use the programs, but they are the supporting modules that will be installed in the conda environment when user installs the deepLearningToolkit. Then, the most safe procedure is to install those versions in what programs was coded and tested.

In fact, this was the main reason to go for virtual environments. Because different xmipp programs was using different versions of these dependencies and they collide...